grunt/config/copy.js

```diff
@@ -1,240 +1,207 @@
+    destFolder = getUnixPath(destFolder);
+    srcFileName = getUnixPath(srcFileName);
 
-        // ignore if the srcFileName ends with the collateAtFolderName
-        var nameParts = srcFileName.split("/");
-        if (nameParts[nameParts.length-1] === collateAtFolderName) {
-            return destFolder;
-        }
+    // ignore if the srcFileName ends with the collateAtFolderName
+    var nameParts = srcFileName.split("/");
+    if (nameParts[nameParts.length - 1] === collateAtFolderName) {
+      return destFolder;
+    }
 
-        var startOfCollatePath = srcFileName.indexOf(collateAtFolderName) + collateAtFolderName.length + 1;
-        var collatedFilePath = destFolder + srcFileName.substr(startOfCollatePath);
+    var startOfCollatePath = srcFileName.indexOf(collateAtFolderName) + collateAtFolderName.length + 1;
+    var collatedFilePath = destFolder + srcFileName.substr(startOfCollatePath);
 
-        return collatedFilePath;
-    }
+    return collatedFilePath;
+  }
 
 
-    var nonServerTasks = {
-        courseAssets: {
-            files: [
-                {
-                    expand: true,
-                    src: ['<%= languages %>/**/*', '!**/*.<%= jsonext %>'],
-                    cwd: '<%= sourcedir %>course/',
-                    dest: '<%= outputdir %>course/'
-                }
-            ]
-        },
-        courseJson: {
-            files: [
-                {
-                    expand: true,
-                    src: ['<%= languages %>/*.<%= jsonext %>'],
-                    cwd: '<%= sourcedir %>course/',
-                    dest: '<%= outputdir %>course/'
-                }
-            ]
-        }
-    };
+  var nonServerTasks = {
+    courseAssets: {
+      files: [{
+        expand: true,
+        src: ['<%= languages %>/**/*', '!**/*.<%= jsonext %>'],
+        cwd: '<%= sourcedir %>course/',
+        dest: '<%= outputdir %>course/'
+      }]
+    },
+    courseJson: {
+      files: [{
+        expand: true,
+        src: ['<%= languages %>/*.<%= jsonext %>'],
+        cwd: '<%= sourcedir %>course/',
+        dest: '<%= outputdir %>course/'
+      }]
+    }
+  };
 
-    var mandatoryTasks = {
-        index: {
-            files: [
-                {
-                    expand: true,
-                    src: ['<%= sourcedir %>index.html'],
-                    dest: '<%= outputdir %>',
-                    filter: 'isFile',
-                    flatten: true
-                }
-            ]
+  var mandatoryTasks = {
+    index: {
+      files: [{
+        expand: true,
+        src: ['<%= sourcedir %>index.html'],
+        dest: '<%= outputdir %>',
+        filter: 'isFile',
+        flatten: true
+      }]
+    },
+    coreAssets: {
+      files: [{
+        expand: true,
+        src: ['<%= sourcedir %>core/assets/**'],
+        dest: '<%= outputdir %>assets/',
+        rename: _.partial(collate, "assets")
+      }]
+    },
+    componentAssets: {
+      files: [{
```

Comment:
@chetan-hajare - Could you put the opening '{' and associated closing '}' on a new line please?